projects/pc/windows/visual_studio/aws_tests/aws_tests.vcxproj

```diff
@@ -1,498 +1,502 @@
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\version.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\version_features.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509_create.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509_crl.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509_crt.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509_csr.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509write_crt.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\x509write_csr.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\library\xtea.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\mbedtls\utils\mbedtls_utils.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\unity\src\unity.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\unity\extras\fixture\src\unity_fixture.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborpretty.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborpretty_stdio.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborencoder.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborencoder_close_container_checked.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborerrorstrings.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborparser.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\tinycbor\cborparser_dup_string.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\http_parser\http_parser.c" />
-    <ClCompile Include="..\..\..\..\..\libraries\3rdparty\jsmn\jsmn.c" />
-  </ItemGroup>
-</Project>
\ No newline at end of file
+	<ItemGroup Label="ProjectConfigurations">
+		<ProjectConfiguration Include="Debug|Win32">
+			<Configuration>Debug</Configuration>
+			<Platform>Win32</Platform>
+		</ProjectConfiguration>
+		<ProjectConfiguration Include="Debug|x64">
+			<Configuration>Debug</Configuration>
+			<Platform>x64</Platform>
+		</ProjectConfiguration>
+	</ItemGroup>
+	<PropertyGroup Label="Globals">
+		<VCProjectVersion>15.0</VCProjectVersion>
+		<ProjectGuid>{D2737FCB-E8AB-49CA-86AD-D6AB594D428E}</ProjectGuid>
+		<Keyword>Win32Proj</Keyword>
+		<RootNamespace>aws_IoT_MCU_Full_Tests</RootNamespace>
+		<ProjectName>aws_tests</ProjectName>
+		<WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+	</PropertyGroup>
+	<Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props"/>
+	<PropertyGroup Condition="&apos;$(Configuration)|$(Platform)&apos;==&apos;Debug|Win32&apos;" Label="Configuration">
```

Comment:
We're all these single quote to &apos; edits intentional? What happened there?

Reply:
These were done as part of the IDE project generation. I think these would be done by some updates to the IDE generator. I have verified that this project builds and runs. So I think these should be harmless.